### DISCUSSION TITLE
Never seen this Icon before.

### DISCUSSION THREAD

**Discussion by GrandpaHardcore** [Hekili/hekili#1400] [Q&A] [answered]
I play a Boomkin (Venthyr) and have the Primordial Arcanic Pulsar legendary on and the rotation seems fine and then out of the blue it shows Starsurge with a small red X in the middle of the icon.  I just default to the next line in the rotation to avoid it because it'll come up even when Starsurge is not ready to be used and am wondering what this Red X is because I've never seen it before on any of my characters.

**Comment by Hekili** [comment, marked as answer]
One feature that people don't know about is, when you don't recognize what is being recommended or you want to know why it is recommended, you can pause the addon and put your mouse over the icon.  (Obviously, not a great plan mid-raid or mid-key, but you can check things out on a dummy later.)  The default keybinding is ALT-SHIFT-P but you can also change this on the /hekili > Toggles tab.

What you are seeing is the recommendation to cancel the Primordial Arcanic Pulsar buff.  The explanation that would show when you pause and mouseover is as follows:

`Cancel the Pulsar buff if the proc wouldn't line up with the next time Convoke comes off cd. This usually only happens during the BL.`

**Comment by Foraga88** [comment]
This seems similar to the issue I posted for Kyrian moonkin. The issue poster is a Venthyr moonkin, so does not have Convoke to use in order to line up with Pulsar
